src/store/modules/rewards/index.ts

```diff
@@ -13,6 +13,10 @@ import { shrinkToken } from "@/api/eth/helpers";
 import { expandToken } from "@/api/pureHelpers";
 import { zip } from "lodash";
 
+const oneMillion = new BigNumber(1000000);
+const ppmToBigNumber = (ppm: number | string): BigNumber =>
+  new BigNumber(ppm).dividedBy(oneMillion);
+
```

Comment:
Let's use use the ```ppmToDec``` then convert that to number, don't need to redefine a oneMillionVariable and the div stuff.